firefoxThemes/waifus/index.html

```diff
@@ -11,6 +11,8 @@
     <script type="module" src="tab.js" defer></script>
     <!--Adds/Removes Waifu-->
     <script src="waifu.js" defer></script>
+  <!--Load Themed Favicon-->
+  <script type="module" src="faviconLoader.js" defer></script>
```

Comment:
Separated favicon theming to a separate file for modularity *and easy testing*.

Can switch out `defer` here for `async` to change the favicon theme lazily. *Preferred*